Engine/Results/BaseResultsHandler.cs

```diff
@@ -316,7 +314,7 @@ public virtual string SaveLogs(string id, List<LogEntry> logs)
         /// <param name="result">The results to save</param>
         public virtual void SaveResults(string name, Result result)
         {
-            File.WriteAllText(GetResultsPath(name), JsonConvert.SerializeObject(result, Formatting.Indented));
```

Comment:
I don't see the harm in leaving this indented for ease of use locally? Why did you choose to remove?

Reply:
So regression tests are as fast as possible, I'll revert the change here and only apply it for the `RegressionResultHandler` 👍